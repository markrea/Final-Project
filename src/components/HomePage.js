import React, { useState } from "react";
import "../styles/App.css";

const HomePage = () => {
  return (
    <div>
      <header className="App-header">
        <h1>Fridge Tracker</h1>
      </header>
    </div>
  );
};

export default HomePage;
