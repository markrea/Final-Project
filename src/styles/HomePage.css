h2{
    color: rgb(68, 67, 67);
    text-align: center;
    padding: 2rem;
    font-family: 'Josefin Slab', serif;
    background: rgba(173, 152, 152, 0.507);
    border-radius: 5px;
    margin: 10px;
    
}
.mainText h4:nth-child(2n+1){
    font-family: 'Josefin Slab', serif;
    background: rgba(214, 203, 203, 0.507);
    margin: 10px;
    border-radius: 10px;
    width: 80%;
    margin: 0 auto;
    margin-bottom: 15px;
}
.mainText h4:nth-child(2n){
    font-family: 'Josefin Slab', serif;
    background: rgba(182, 159, 159, 0.507);
    margin: 10px;
    border-radius: 10px;
    width: 80%;
    margin: 0 auto;
    margin-bottom: 15px;
}

.mainText h4{
    color: rgb(73, 71, 71);
}
