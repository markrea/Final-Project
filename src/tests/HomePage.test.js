import React from "react";
import { render } from "@testing-library/react";
import HomePage from "../components/HomePage";
import "@testing-library/jest-dom/extend-expect";

describe("HomePage", () => {
  it("renders component", () => {
    const { asFragment } = render(<HomePage />);
    expect(asFragment).toMatchSnapshot();
  });
});
