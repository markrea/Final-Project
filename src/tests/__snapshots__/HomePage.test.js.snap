// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders component 1`] = `[Function]`;
